cyb_oko/settings.py

```diff
@@ -40,16 +40,15 @@
     'django_filters',
 )
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
     'corsheaders.middleware.CorsMiddleware',
-    'cyb_oko.querydebug.QueryCountDebugMiddleware',
+    #'cyb_oko.querydebug.QueryCountDebugMiddleware',
```

Comment:
Fjern denne om man ikke bruker den lenger

Reply:
Fant ikke noe sted det var brukt før heller, den var installert, men ikke brukt til noe så vidt jeg kunne se. Isåfall fjerner vi hele querydebug filen også. Var noe der som ikke fungerte med django2*